Vagrantfile

```diff
@@ -161,7 +161,7 @@ Vagrant.configure("2") do |config|
     v.linked_clone = true
     v.memory = 2048
     v.cpus = 2
-    v.customize ["modifyvm", :id, "--chipset", "ich9"]
+    v.customize ["modifyvm", :id, "--chipset", "ich9", "--audio", "none"]
```

Comment:
What's the relation between `audio` and the length of the name?

Reply:
I don't really know, but it works.
I tried to find more information on the Internet on why it's related, but didn't manage to find much.
It's not directly related to the VM name as far as I understand, check this comment: https://github.com/hashicorp/vagrant/issues/9524#issuecomment-389269817 (the only relevant thing I found).

Reply:
I'll update the commit message with more precise information.